### PR TITLE
Add hint to README.md when running Jenkins as a docker container

### PR DESCRIPTION
**Pull request by friggingee**
Singed-off-by: Grigori Prokhorov <hello@feinberg.it>

### CODE REVIEW

File: integrations/jenkins/README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Please follow the regular [installation instructions](https://www.jenkins.io/doc
 
 Also ensure that [Docker is installed](https://docs.docker.com/engine/install/) on your system with [BuildKit enabled](https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds), and that Jenkins is allowed to run Docker (you might need to run `usermod -aG docker jenkins` to add the `jenkins` user to the `docker` group).
 
+If you're running Jenkins as a docker image as opposed to using the Java WAR, you need to [install docker inside the Jenkins container](https://docs.docker.com/engine/install/debian/).
```
Comment (sschuberth) [inline integrations/jenkins/README.md:15]: Nits:
- In docs, we prefer to spell out "you're" as "you are".
- "docker" -> "Docker" (two times).
- You're referring to Debian-specific install instructions, but we should stay OS-agnostic as much as possible. So probably just use https://docs.docker.com/engine/install/ (without the trailing "debian/") instead.

**Review by sschuberth — CHANGES_REQUESTED** [review]
Please amend your existing commit in this PR with the requested changes instead of creating another commit.

File: integrations/jenkins/README.md (hunk 1 above)
Comment (friggingee) [reply integrations/jenkins/README.md:15]: 1 and 2 are fine but 3 relates to the fact that the official Jenkins Docker images use Debian specifically. 
Jenkins has always provided docker images based on Debian from the oldest available images on Docker hub.
I'd argue this is more user friendly?

Comment (sschuberth) [reply integrations/jenkins/README.md:15]: > Jenkins has always provided docker images based on Debian

Ok, makes sense then!

**Comment by friggingee** [comment]
I need to redo this PR cleanly. Sorry, closing this.

**Pull request closed without merging**